### PR TITLE
Resolve: Create migration for drop table if a entire schema is deleted from OpenAPI spec

### PR DESCRIPTION
**Pull request by SOHELAHMED7**
Fix https://github.com/cebe/yii2-openapi/issues/132

Fix https://github.com/php-openapi/yii2-openapi/issues/2

PR in fork: https://github.com/SOHELAHMED7/yii2-openapi/pull/29

### CODE REVIEW

**Comment by SOHELAHMED7** [comment]
Changing status to Draft to clean up few unwanted changes

**Comment by SOHELAHMED7** [comment]
Changing status to Ready for review

File: src/lib/generators/MigrationsGenerator.php [hunk 1]
```diff
@@ -147,7 +152,6 @@ protected function createBuilder(DbModel $model):BaseMigrationBuilder
     protected function sortMigrationsByDeps():array
     {
         $this->sorted = [];
-        ksort($this->migrations);
```
[the remark this answers is not in the document]
Comment (SOHELAHMED7) [reply src/lib/generators/MigrationsGenerator.php:150]: If multiple tables are dropped at same time then it is used. But as mentioned above I will revert for non-dependent multiple tables drop scenario.

[the remark this answers is not in the document]
Comment (SOHELAHMED7) [reply src/lib/generators/MigrationsGenerator.php:150]: Implementation of this feedback is complete.

**Comment by cebe** [comment]
please resolve conflicts with master.

**Review by cebe — APPROVED** [review]

**Comment by cebe** [comment]
Thank you!

**Pull request merged**